examples/demo/src/visitors/VisitorCreate.js

```diff
@@ -28,29 +31,20 @@ const VisitorCreate = ({ classes, ...props }) => (
     <Create {...props}>
         <TabbedForm>
             <FormTab label="resources.customers.tabs.identity">
-                <TextInput
-                    autoFocus
-                    source="first_name"
-                    formClassName={classes.first_name}
-                />
-                <TextInput
-                    source="last_name"
-                    formClassName={classes.last_name}
-                />
+                <TextInput autoFocus source="first_name" formClassName={classes.first_name} />
+                <TextInput source="last_name" formClassName={classes.last_name} />
                 <TextInput
                     type="email"
                     source="email"
                     validation={{ email: true }}
                     fullWidth={true}
                     formClassName={classes.email}
                 />
-                <DateInput source="birthday" />
+                <DateInput source="birthday" formClassName={classes.birthday} />
+                <PasswordInput source="password" formClassName={classes.password} />
```

Comment:
Since this is a demo, we should showcase how to hav two password fields and a validator that makes sure the two are correct